.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -0,0 +1,50 @@
+param ($GitHub)
+
+switch ($GitHub.event_name)
```

Comment:
Don't you need a default case for this? Or to check if `$summary` is null or empty at the end before creating outputs.

Reply:
I knew you'll notice this but I was like "eh, who calls this for some other event deserves their fate".